cryodrgn/commands/filter.py

```diff
@@ -80,13 +81,19 @@ def add_args(parser: argparse.ArgumentParser) -> None:
         help="path to a file containing previously selected indices "
         "that will be plotted at the beginning",
     )
+    parser.add_argument(
+        "--sel-dir",
```

Comment:
Maybe we should have a more general `--save-location` / `-s` argument that can take either a filename or a directory, and will save the selection there if given a filename without further prompting (like `--force`) and behave like this `--sel-dir` if a directory is given?